<launch>

    <arg name="namespace" default="" doc="Specify the name as parent_to_child"/>
    <arg name="parent_frame" default="" doc="Specify the parent frame name"/>
    <arg name="child_frame" default="" doc="Specify the child frame name"/>
    <arg name="freq" default="15.0" doc="Specify the child frame name"/>



    <group ns="$(arg namespace)">
        
        <param name="parent_frame" type="str" value="$(arg parent_frame)"/>
        <param name="child_frame" type="str" value="$(arg child_frame)"/>
        <param name="freq" type="double" value="$(arg freq)"/>
        <node name="twist_tf_info_node" pkg="zerog_utilities" type="twist_tf_info_node"/>

    </group>



</launch>